### ISSUE TITLE
API limits for scraping?

### ISSUE THREAD

**Issue by nikshepsvn** [jmfernandes/robin_stocks#198]
I want to pull a ton of option data but I'm afraid I might get my account flagged, anyone have experience with scraping?

**Comment by edukaded** [comment]
Yeah, don't do it.  It sucks.  It's more than a flag, it's a ban if you happen to not be using the same IP as your phone.
Best way to do something like options is to select a handful of stocks you're really interested in, max of like 10, then choose dates within the quarter.  If you need deeper, per option like info like greeks make sure you're using something like async / await and add at least 1 second plus a random amount of extra wait.  This will keep you from getting rate limited / flagged / banned for the most part. 
https://docs.python.org/3/library/asyncio-task.html
